### DISCUSSION TITLE
toolbar option

### DISCUSSION THREAD

**Discussion by fredleput** [megahertz/react-simple-wysiwyg#9] [Q&A]
Hi,
First thx for your work !
I was wondering if it was possible to remove buttons from the toolbar ?
I tried something like this but it doesn't work...

```
<DefaultEditor 
  value= {formAddState.depot.commentaires}
  toolbar= {{
    options: ["here the list of needed buttons"]
  }}
  onChange={onEditorChange}
/>
```
Thx for reading,
Fred, Brittany - France

**Comment by megahertz** [comment]
Hi Fred.

You need to use `Editor` instead of `DefaultEditor` to configure toolbar, `DefaultEditor` is just a facade. So, your code may [look like that](https://github.com/megahertz/react-simple-wysiwyg/blob/master/src/editor/DefaultEditor.tsx).

**Reply by alefebspp** [reply]
One question: Can i style those individual components of the example with tailwindcss?

**Reply by megahertz** [reply]
> One question: Can i style those individual components of the example with tailwindcss?

Unfortunately no, most of components have a fixed className property

**Reply by alefebspp** [reply]
Ok. I've tried to implement it following the example on the npm page, but for some reason the list style for the editor is'nt working. Any idea why?

**Reply by megahertz** [reply]
> Ok. I've tried to implement it following the example on the npm page, but for some reason the list style for the editor is'nt working. Any idea why?

If you can share an example which shows the bug I can take a look
